src/hooks/events.ts

```diff
@@ -109,7 +109,7 @@ export class AfterToolCallEvent extends HookEvent {
     input: JSONValue
   }
   readonly tool: Tool | undefined
-  readonly result: ToolResultBlock
+  result: ToolResultBlock
```

Comment:
Separate out writable properties by moving them lower below the readonly properties.